example/otel-collector/main.go

```diff
@@ -122,6 +122,11 @@ func main() {
 	meter.RegisterView(metric.NewView(vr.SyncImpl(), metric.LabelKeys, []label.Key{"os_type"}, nil))
 	meter.RegisterView(metric.NewView(vr.SyncImpl(), metric.LabelKeys, []label.Key{"environment"}, nil))
 
+	// User defined histograms view, histograms will be aggregated with users defined buckets.
+	customBuckets := []float64{1.0, 2.0, 3.0, 4.0}
+	histAggregatorFactory := NewHistogramAggregatorFactory(WithExplicitBoundaries(customBuckets), WithDescriptor(vr.SyncImpl().Descriptor()))
```

Comment:
could u make it not dependent on the `vr.SyncImpl().Descriptor()`?

Reply:
the ideal interface would be `NewHistogramAggregatorFactory(bound_argument)`.

Reply:
The inbuilt histogram aggregator requires the metrics descriptor to determine the `numberKind` which is used to construct the aggregator. Not sure if I can remove this dependency, did you mean something else?

Reply:
> the ideal interface would be NewHistogramAggregatorFactory(bound_argument).

I think I could accomplish that by instantiating the aggregator with a placeholder metric descriptor with the numberkind as float or something